relay-server/src/actors/global_config.rs

```diff
@@ -148,8 +149,9 @@ pub struct GlobalConfigService {
 impl GlobalConfigService {
     /// Creates a new [`GlobalConfigService`].
     pub fn new(config: Arc<Config>, upstream: Addr<UpstreamRelay>) -> Self {
-        let (global_config_watch, _) = watch::channel(Arc::default());
         let (internal_tx, internal_rx) = mpsc::channel(1);
+        let (global_config_watch, _) = watch::channel(Arc::new(GlobalConfig::default()));
```

Comment:
`Arc::default` already handles the default, see [docs](https://doc.rust-lang.org/std/sync/struct.Arc.html#impl-Default-for-Arc%3CT%3E):

> Creates a new Arc<T>, with the Default value for T.